### PR TITLE
Added 2 more test cases to util.test.ts

### PR DESCRIPTION
**Pull request by Dhanush-Reddy**

### CODE REVIEW

File: __tests__/util.test.ts [hunk 1]
```diff
@@ -207,6 +207,19 @@ describe('isValidURL', () => {
...
+    ['https://github.com/docker/buildx.git', false],
+    ['https://github.com/docker/buildx.git#refs/pull/648/head', false],
+    ['git@github.com:moby/buildkit.git', true],
+    ['git://github.com/user/repo.git', true],
+    ['github.com/moby/buildkit.git#main', true],
+    ['v0.4.1', true]
+  ])('given %p', async (url, expected) => {
+    expect(Util.isValidURL(url)).toEqual(expected);
+  });
+});
```
Comment (crazy-max) [inline __tests__/util.test.ts:221]: This is already covered by `isValidURL`

File: __tests__/util.test.ts [hunk 2]
```diff
@@ -220,6 +233,19 @@ describe('isValidRef', () => {
...
+    ['https://github.com/docker/buildx.git', false],
+    ['https://github.com/docker/buildx.git#refs/pull/648/head', false],
+    ['git@github.com:moby/buildkit.git', false],
+    ['git://github.com/user/repo.git', false],
+    ['github.com/moby/buildkit.git#main', false],
+    ['v0.4.1', true]
+  ])('given %p', async (url, expected) => {
+    expect(Util.isValidRef(url)).toEqual(expected);
+  });
+});
```
Comment (crazy-max) [inline __tests__/util.test.ts:247]: Same here with `isValidRef`

**Comment by crazy-max** [comment]
Closing for now but let us know if you have other questions thx.

File: __tests__/util.test.ts (hunk 1 above)
Comment (Dhanush-Reddy) [reply __tests__/util.test.ts:221]: hey max i am new to open sourcing went ahead just open the repo and tried to contribute something to the community 
i know my code won't any new ground breaking chages or bring anything i did cause it it wont change anything.
If possible merge it or ill try to contribute for some other thing

Comment (crazy-max) [reply __tests__/util.test.ts:221]: > hey max i am new to open sourcing went ahead just open the repo and tried to contribute something to the community

That's perfectly fine and thanks for taking a step in the open source community and contributing to this project!

> i did cause it it wont change anything.
> If possible merge it or ill try to contribute for some other thing

Well that's actually an issue. If your change does not have any purpose but just for the sake of adding more code, it won't be approved. It needs to have a relevant purpose (bug fix, enhancement, docs, ...).

Comment (Dhanush-Reddy) [reply __tests__/util.test.ts:221]: I actually went there and reached out to bunch of people every one is busy with their own work and some are charging way to high as your also an developer in the space i thought u could help that it Max nothing else

**Pull request closed without merging**